<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WAccountPageClass</class>
 <widget class="QWidget" name="WAccountPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WAccountPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font: 16px &quot;微软雅黑&quot;;
}

QFrame#frame_total{
	border-radius:6px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 102, 255, 255), stop:1 rgba(189, 0, 255, 255));
}

QFrame#frame_name{
	background-color: #E5F0FF;
	border:none;
	border-radius:6px
}

QFrame#frame_user{
	background-color: #F0FFFA;
	border:none;
	border-radius:6px
}

QFrame#frame_account{
	background-color: #F0FFFA;
	border:none;
	border-radius:6px
}

QFrame#frame_phone{
	background-color: #E5F0FF;
	border:none;
	border-radius:6px
}

QPushButton#pb_charge{
	background-color: rgb(255, 255, 255,0);
	color: rgb(255, 255, 255);
	border-radius:6px;
	border: 1px solid #ffffff;
	font: 14px &quot;微软雅黑&quot;;
}

QLabel#lb_9{
	color: rgb(255, 255, 255);
}

QLabel#lb_time{
	color: rgb(255, 255, 255);
	font: 14px &quot;微软雅黑&quot;;
}



</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>16</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>25</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_name">
           <property name="minimumSize">
            <size>
             <width>192</width>
             <height>62</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>100</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>18</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Mobilephone</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_phone">
              <property name="text">
               <string>12345778</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_user">
           <property name="minimumSize">
            <size>
             <width>192</width>
             <height>62</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>199</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>18</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>UserName</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_name">
              <property name="text">
               <string>Jone joe</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>25</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_account">
           <property name="minimumSize">
            <size>
             <width>192</width>
             <height>62</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>192</width>
             <height>62</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>18</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>AccountNumber</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_account">
              <property name="text">
               <string>#35666</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_phone">
           <property name="minimumSize">
            <size>
             <width>192</width>
             <height>62</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>192</width>
             <height>62</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>18</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Total Points</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_point">
              <property name="text">
               <string>23455</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame_total">
         <property name="minimumSize">
          <size>
           <width>409</width>
           <height>94</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>140</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="lb_9">
            <property name="text">
             <string>Package Status</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="lb_time">
              <property name="minimumSize">
               <size>
                <width>190</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Premium(Expires on 12/05/2023)</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pb_charge">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>35</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>Recharge</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>83</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>83</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
